mitreattack/navlayers/core/exceptions.py

```diff
@@ -105,3 +105,21 @@ def loadChecker(caller, testee, required, field):
         if entry not in testee:
             handler(caller, '{} is not present in {} [{}]'.format(entry, field, testee))
             raise MissingParameters
+
+
+def handle_object_placement(handle_to_self_field, potential_object, objectType, list=False):
+    """
+        If the input element (potential_object) is an object of objectType, assign it to handle_to_self_field
+        :param handle_to_self_field: the field the object is to be assigned to
+        :param potential_object: the potential instance of an object
+        :param objectType: the type of object being looked for
+        :param list: whether or not handle_to_self_field should be a list
+        :return: bool on whether or not the object was assigned to handle_to_self_field
```

Comment:
Return type doesn't properly document the case where the field was assigned. As far as I can see it only returns bool if it was not able to be assigned.

Reply:
Yep, looks like I didn't update the documentation from the previous version, which only returned bools and tried to do setting by reference (couldn't get it to work quite right, so I changed the approach).